readme.md

```diff
@@ -35,16 +35,17 @@ unusedFilename('rainbow.txt').then(filename => {
 
```

Comment:
When an API uses promises, you could also update the example to use async/await:

```js
(async () => {
	console.log(await unusedFilename('rainbow.txt'));
	//=> 'rainbow (2).txt'
})();
```